src/main/java/io/hypermine/hypersign/credentails/Credential.java

```diff
@@ -0,0 +1,10 @@
+package io.hypermine.hypersign.credentails;
+
+public interface Credential {
```

Comment:
Just one suggestion. Can you please name interfaces with prefix `I`. like `ICredential`? and then class that implements this interface would be `Credential`. This will help in simplicity and understanding.